pkg/api/pluginproxy/pluginproxy.go

```diff
@@ -22,10 +22,10 @@ type templateData struct {
 
 // NewApiPluginProxy create a plugin proxy
 func NewApiPluginProxy(ctx *models.ReqContext, proxyPath string, route *plugins.Route,
-	appID string, cfg *setting.Cfg, secretsService secrets.Service) *httputil.ReverseProxy {
+	appID string, cfg *setting.Cfg, store sqlstore.Store, secretsService secrets.Service) *httputil.ReverseProxy {
```

Comment:
Nit. An alternative to this was a trick I recently used by defining a local package interface for only those methods that needs to be mocked, for example 
https://github.com/grafana/grafana/blob/main/pkg/services/plugindashboards/plugindashboards.go#L15-L17
https://github.com/grafana/grafana/blob/61533a3cb4fbacc3071b4e5a64a8f366b5ae36cc/pkg/services/plugindashboards/plugindashboards_test.go#L355-L365

Reply:
Exactly, thanks! That's what I've been doing with the alerting services and their stores. But when it comes to pkg/api - we decided to proceed with a huge Store interface everywhere, so that we could later "grep" for it during the following store splitting phase. Thus, rather than creating dozens of interfaces that overlap with each other - we keep Store for now to decide how to better split it into smaller, independent stores later.